### DISCUSSION TITLE
Issues with Alex and Vivaldi

### DISCUSSION THREAD

**Discussion by ducky1241** [AdguardTeam/AdGuardHome#7137] [Q&A]
I am running v0.107.52 on my Windows 11 server. I have worked out most of the "kinks'. Unfortunately I am having two small issues.
 1.  As soon as I enable Adguard all of my Alexa devices go offline. I had tried using different blocklists, nothing seems to work. Can someone tell me if there is something I am missing.
 2. I use Vivaldi as my default browser but Adguard doesn't seem to be blocking any of the ads. I have no issues with Firefox and Chrome. Anyone have any ideas how I can fix this?

**Comment by Dynamic5912** [comment]
1. Have you tried disabling all blocklists?

2. Check for SecureDNS/DOH being enabled in the browser settings

**Reply by ducky1241** [reply]
1. if I disable the blocklist, what is the point of using Adguard.\
2.  Did that, doesn't do anything.

**Reply by Dynamic5912** [reply]
> 1. if I disable the blocklist, what is the point of using Adguard.\
> 
> 2.  Did that, doesn't do anything.
> 
> 

To test if the Alexa's still work. Do they work with no blocklists enabled?

**Reply by ducky1241** [reply]
Alex still works with no blocklist
I have tried to whitelist the servers it uses, but doesn't help

**Reply by Dynamic5912** [reply]
> Alex still works with no blocklist
> 
> I have tried to whitelist the servers it uses, but doesn't help

Then there is an issue with your setup. Especially if your browser isn't getting ads blocked either

**Reply by ducky1241** [reply]
Obviously there is an issue. That's why I posted here for suggestions. Only Vivaldi is having trouble, All my other browsers work

**Reply by Dynamic5912** [reply]
> Obviously there is an issue. That's why I posted here for suggestions. Only Vivaldi is having trouble, All my other browsers work

What does a DNS leak test show from Vivaldi?

Is it the same DNS upstream servers you have set in AGH?
